Portals/_default/Skins/Vanjaro/Base.ascx.cs

```diff
@@ -845,6 +846,32 @@ private void LoadModuleInFrame()
             }
         }
 
+
+        private void HomeLogoAndSiteIcons()
+        {
+            string PortalRoot = PageManager.GetPortalRoot(PortalSettings.Current.PortalId);
+            PortalRoot = PortalRoot + "/";
+            string SocialSharingLogo = PortalRoot + PortalController.GetPortalSetting("SocialSharingLogo", PortalSettings.Current.PortalId, "", PortalSettings.CultureCode);
+            string HomeScreenIcon = PortalRoot + PortalController.GetPortalSetting("HomeScreenIcon", PortalSettings.Current.PortalId, "", PortalSettings.CultureCode);           
+            Literal lt = new Literal();
+            if (!string.IsNullOrEmpty(SocialSharingLogo))
+            {
+                lt.Text += "<meta property=\"og:image\" content=\"" + SocialSharingLogo + "\">";
+                lt.Text += "<meta property=\"og:image:type\" content=\"image/png\">";
+                lt.Text += "<meta property=\"og:image:width\" content=\"1024\">";
+                lt.Text += "<meta property=\"og:image:height\" content = \"1024\">";
+            }
+
+            if (!string.IsNullOrEmpty(HomeScreenIcon))
+            {
+                lt.Text += "<link rel=\"icon\" type=\"image/png\" sizes=\"64x64\" href=\"" + HomeScreenIcon + "\">";
+                lt.Text += "<link rel=\"apple-touch-icon\" sizes=\"180x180\" href=\"" + HomeScreenIcon + "\">";
+            }
```

Comment:
We're using the same image with different sizes. Doesn't make sense. Are we requesting the larger size at upload?